sdk/samples/ci.yml

```diff
@@ -0,0 +1,34 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+    - sdk/samples
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+    - sdk/samples
+
+stages:
+- template: /eng/pipelines/templates/jobs/archetype-sdk-client.yml
```

Comment:
I think this should be archetype-sdk-client-samples.yml

Reply:
That looks like it might be better. I'd like to add in tests too but this is better than what's going on now.